lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -115,7 +140,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
   /// [PlatformWindow.scheduleFrame] has been called since the last time this
   /// callback was invoked.
   @override
-  ui.FrameCallback? get onBeginFrame => _onBeginFrame;
+  ui.FrameCallback? get onBeginFrame => onBeginFrame;
```

Comment:
@ditman  haha of course this is an infinite loop.

Reply:
Only Chuck Norris would have been able to finish executing `onBeginFrame`! 😭